### ISSUE TITLE
Action button for Interaction with environment

### ISSUE THREAD

**Issue by wushin** [tales/tales-client#37]
talk to NPCs, Search Chests, etc.

**Comment by bjorn** [comment]
Agreed, maybe a button that appears when you're getting near. The pressing on the actual NPC does not feel very nice and isn't clear.
